File: Acwing/2-Prac-1-Week Game/29-week/4194. pow/Main.cpp
#include<iostream>
#include<cmath>
using namespace std;

const double basic = 1.00011;

double methodOne(int n , int t);

int main(){
    int n , t;
    cin >> n >> t;
    double res = methodOne(n , t);
    printf("%6lf",res);
    return 0;
}

double methodOne(int n , int t){
    return n * pow(basic , (double)t);
}

File: Acwing/2-Prac-1-Week Game/29-week/4194. pow/Main.java
import java.util.Scanner;

public class Main {

    static int n , t ;

    static double res;

    static final double basic = 1.00011;
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();
        t = sc.nextInt();
        res = methodOne(n, t);
        System.out.printf("%6f", res);
    }

    public static double methodOne(int n , int t){
        return n * Math.pow(basic, t);
    }
}
